getting_started.rst

```diff
@@ -63,7 +63,8 @@ If you don't use conda (see above), create a virtual environment::
     Depending on your platform, the Python 3 interpreter could be invoked by
     ``python`` instead. This is the case for conda on Windows for example.
 
-Install ``aiohttp`` in the virtual environment::
+Activate the virtual environment and install ``aiohttp`` inside:
 
-    ./venv/bin/python -m pip install -U aiohttp
+    source ./venv/bin/activate
+    pip install -U aiohttp
```

Comment:
In this case, it'd be good to mention that users can test their installation using:

    $ venv/bin/python examples/hello_world.py
    hello world

to make it clear that they have to use `venv/bin/python` instead of `python` or `python3` to run the examples.